### ISSUE TITLE
Vitest workspaces not properly supported by `defineVitestConfig`

### ISSUE THREAD

**Issue by TheDutchCoder** [nuxt/test-utils#1043]
### Environment

```
------------------------------
- Operating System: Darwin
- Node Version:     v20.13.0
- Nuxt Version:     3.14.1592
- CLI Version:      3.16.0
- Nitro Version:    2.10.4
- Package Manager:  npm@10.5.2
- Builder:          -
- User Config:      default
- Runtime Modules:  @nuxt/eslint@0.7.2, nuxt-open-fetch@0.9.5, @nuxtjs/i18n@9.1.0, nuxt-jsonld@2.1.0, @nuxtjs/device@3.2.4, @nuxtjs/robots@5.0.0, @vueuse/nuxt@12.0.0, @sidebase/nuxt-auth@0.9.4, @vesp/nuxt-fontawesome@1.1.0
- Build Modules:    -
------------------------------
```

### Reproduction

When using a vitest workspace, vitest complains that it can't resolve the config files, this is due to `defineVitestConfig`.
```
> vitest

failed to load config from /Users/reinierkaper/rvezy/rvezy-web-client/src/tests/unit/vitest.config.ts
```

### Describe the bug

I think the `defineVitestConfig` wrapper conflicts with what vitest expects back in a workspace configuration file.
Everything works fine when the tests are ran directly with a config file reference, just not in a workspace.

### Logs

```shell-script

```

**Comment by gaisinskii** [comment]
I was going to raise a similar issue, but since this one exists I will ask my question here.

TLDR:

1) Open https://stackblitz.com/~/github.com/gaisinskii/nuxt-split-tests-issue
2) Wait for deps installation
3) Run `pnpm run test:unit` or `pnpm run test:e2e`
4) Console says `No test files found. You can change the file name pattern by pressing "p"`

The issue:

My colleagues and I want to separate unit & e2e tests and from what I understand the only way of doing it is to use  [vitest workspaces](https://vitest.dev/guide/workspace#configuration). I've tried different configurations and the only way it worked was to replace `defineVitestConfig` with plain `defineConfig`.

**Comment by danielroe** [comment]
this is now resolved
